### PR TITLE
added change in revocations slider

### LINKED ISSUE #5: Implement the vertical revocation slider
**Is your feature request related to a problem? Please describe.**
As described in the [PRD](https://docs.google.com/document/d/1vssQ4JB8VyfInFpdw7F_4laQG0WVXYv6u4FwIpIFUoQ/edit?ts=5f58fbbc#heading=h.c29ll7o2tvsa), there are a number of interactive elements used in the component. We should implement the revocation slider as described and mocked.

**Describe the solution you'd like**
This should have the look-and-feel and constraints documented in the mocks. It should move from -100% on the bottom to +100% on the top, with a default value of -50% on new page load. There is an interacting text block that should move alongside the slider, including both the larger "X%" text and the smaller descriptive text that reads, "Y violations resulting in state incarceration." That smaller text does not yet need to be interactively updated, but we should ensure that the correct UX is in place, with that text staying below the larger "X%" text. 

Like the previous task, #4, this does not yet need to be connected to any logic elsewhere, but it should have the desired interaction, e.g. sliding up and down, as described in the PRD.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

- change in revocations slider added

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #5 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/SupervisionSuccess/SupervisionSuccess.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const SupervisionSuccess = () => {
     setProjections(newProjections);
   }, []);
   const onChangeInRevocationsChange = useCallback((newChangeInRevocations) => {
+    console.log(newChangeInRevocations);
```
Comment (jessex) [inline src/SupervisionSuccess/SupervisionSuccess.js:24]: Let's delete this now that we're about to merge.

**Review by jessex — CHANGES_REQUESTED** [review]
Looks and functions to spec. Well done, @Wmaileq!

**Review by jessex — APPROVED** [review]
Went ahead and removed that console.log statement myself, so we're good to go now!

**Comment by Wmaileq** [comment]
@jessex thanks for removing console log, fixed it but forgot to push changes

**Pull request merged**